etna/transforms/scalers.py

```diff
@@ -37,11 +39,16 @@ def __init__(
             if True, center the data before scaling.
         with_std:
             if True, scale the data to unit standard deviation.
+        mode:
+            "macro" or "per-segment", way to transform features over segments.
+            If "macro", transforms features globally, gluing the corresponding ones for all segments.
+            If "per-segment", transforms features for each segment separately.
```

Comment:
May be we should add Raises block here.

Reply:
We don't have Raises block in the similar cases in `_OneSegmentTimeSeriesImputerTransform` or `Metric`

Reply:
Done